context/default/context.go

```diff
@@ -0,0 +1,29 @@
+package defaultctx
+
+import "github.com/runabove/venom"
+
+// Context Type name
+const Name = "default"
+
+// New returns a new TestCaseContext
+func New() venom.TestCaseContext {
+	ctx := &TestCaseContext{}
+	ctx.Name = Name
+	return ctx
+}
+
+// TestCaseContext represents the context of a testcase
+type TestCaseContext struct {
```

Comment:
Can you rename on DefaultTestCaseContext ? (to avoid confusion with Interface name)